tuplex/core/src/DataSet.cc

```diff
@@ -734,10 +726,14 @@ namespace tuplex {
     }
 
 
-    void DataSet::show(const int64_t numRows, std::ostream &os) {
+    void DataSet::show(int64_t numRows, std::ostream &os) {
```

Comment:
why not here size_t then as well? incl. special value? pls. add to doxygen.

Reply:
It includes -1, which means show everything, since python don't usually use size_t::max()